pom.xml

```diff
@@ -59,21 +59,21 @@
         <protobuf.plugin.version>0.6.1</protobuf.plugin.version>
         <compiler.plugin.version>3.8.0</compiler.plugin.version>
 
-        <!-- Dependency Versions -->
-        <reactive.streams.version>1.0.2</reactive.streams.version>
-        <grpc.version>1.23.0</grpc.version>
-        <protoc.version>3.9.0</protoc.version> <!-- Same version as grpc-proto -->
-        <jprotoc.version>0.9.1</jprotoc.version>
-        <rxjava.version>2.2.12</rxjava.version>
-        <reactor.version>3.2.12.RELEASE</reactor.version>
+		<!-- Dependency Versions -->
+		<reactive.streams.version>1.0.2</reactive.streams.version>
+		<grpc.version>1.31.0</grpc.version>
+		<protoc.version>3.12.0</protoc.version> <!-- Same version as grpc-proto -->
+		<jprotoc.version>1.0.1</jprotoc.version>
+		<rxjava.version>3.0.5</rxjava.version>
+		<reactor.version>3.3.8.RELEASE</reactor.version>
```

Comment:
Why are you also upgrading Spring Reactor in an RxJava3 PR?

Reply:
Same as above update of gRPC. Will revert if you want it in separate PR